app/models/photo.rb

```diff
@@ -0,0 +1,11 @@
+class Photo < ActiveRecord::Base
+
+  has_attached_file :image, :styles => { :medium => "640x640>", :thumb => "100x100>" }, :default_url => "/images/:style/missing.png"
```

Comment:
Metrics/LineLength: Line is too long. [132/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)